core/src/main/scala/com/salesforce/op/filters/RawFeatureFilter.scala

```diff
@@ -371,6 +373,10 @@ object RawFeatureFilter {
     bins
   }
 
+  // If there are not enough rows in the scoring set, we should not perform comparisons between the training and
+  // scoring sets since they will not be reliable. Currently, this is set to the same as the minimum training size.
+  val minRowsForScoringSet = 500
```

Comment:
lets not make this hard coded - it should be a parameter that the user can override

Reply:
Good point, fixed.